pkg/integration/tests/tag/crud_lightweight.go

```diff
@@ -41,11 +42,33 @@ var CrudLightweight = NewIntegrationTest(NewIntegrationTestArgs{
 					).
 					PressEscape()
 			}).
+			Press(keys.Universal.Push).
+			Tap(func() {
+				t.ExpectPopup().Prompt().
+					Title(Equals("Remote to push tag 'new-tag' to:")).
+					InitialText(Equals("origin")).
+					SuggestionLines(
+						Contains("origin"),
+					).
+					Confirm()
+			}).
+			Press(keys.Universal.Remove).
+			Tap(func() {
+				t.ExpectPopup().
+					Menu().
+					Title(Equals("Delete tag 'new-tag'?")).
+					Select(Equals("r Delete the remote tag")).
+					Confirm()
+			}).
+			Lines(
+				MatchesRegexp(`new-tag.*initial commit`).IsSelected(),
+			).
 			Press(keys.Universal.Remove).
 			Tap(func() {
-				t.ExpectPopup().Confirmation().
-					Title(Equals("Delete tag")).
-					Content(Equals("Are you sure you want to delete tag 'new-tag'?")).
+				t.ExpectPopup().
```

Comment:
Can we do the local+remote options too (feel free to do it in another integration test cos this one's getting long)